src/nvim/api/win_config.c

```diff
@@ -130,6 +131,14 @@
 ///   - focusable: Enable focus by user actions (wincmds, mouse events).
 ///       Defaults to true. Non-focusable windows can be entered by
 ///       |nvim_set_current_win()|.
+///   - mouseevent: Specify how this window affects mouse events.
```

Comment:
Normally callbacks should be `on_xx`, but if `on_mouse` is too awkward here, we can just call it `mouse`.
```suggestion
///   - mouse: Specify how this window affects mouse events. Defaults to `focusable` value.
```